app/Http/Controllers/RefreshedNftController.php

```diff
@@ -21,8 +20,6 @@ public function __invoke(Collection $collection, Nft $nft): JsonResponse
         // It's supposed to be completely opaque to the user what the "refresh" is doing.
         RefreshNftMetadata::dispatch($collection, $nft)->onQueue(Queues::NFTS);
 
-        FetchNftActivity::dispatch($nft);
```

Comment:
I'd still fetch it for the entire collection so it can be updated on a different interval if needed. This is mainly to allow to change activity to be a weekly scheduled job; then we run the risk of some activities being behind and we should allow the users to update them once a day (1x per day per collection max, irregardless of user requesting it)